ceno_zkvm/src/instructions/riscv/mul.rs

```diff
@@ -128,11 +134,15 @@ pub struct MulhConfig<E: ExtensionField> {
     rs2_read: UInt<E>,
     rd_written: UInt<E>,
     sign_deps: MulhSignDependencies<E>,
-    prod_low: UInt<E>,
     r_insn: RInstructionConfig<E>,
+    /// The low/high part of the result of multiplying two Uint32.
```

Comment:
Thanks!

Could you expand this a bit, like this or similar?:

```rust
/// The low/high part of the result of multiplying two Uint32.
///
/// Whether it's low or high depends on the operation.
```

Reply:
added in  [2114670](https://github.com/scroll-tech/ceno/pull/660/commits/2114670c5281259da340908eddecb07f29d4b608)